docs/patterns/single-new-eks-observability-accelerators/single-new-eks-native-observability.md

```diff
@@ -0,0 +1,137 @@
+# Single New EKS Cluster Native Observability Accelerator
+
+## Architecture
+
+The following figure illustrates the architecture of the pattern we will be deploying for Single EKS Cluster Native Observability pattern using AWS native tools such as CloudWatch Logs and Container Insights:
+
+![Architecture](../images/native-arch.png)
```

Comment:
Image needs a little bit of more details. Please see this diagram https://aws-observability.github.io/observability-best-practices/guides/containers/aws-native/eks/amazon-cloudwatch-container-insights/#:~:text=with%20Fargate%20profile.-,Figure%3A%20Pipeline%20components%20in%20an%20ADOT%20Collector%20instance%20deployed%20to%20Amazon%20EKS,-In%20the%20above and you can use things like metrics pipeline from this to show the ADOT pipeline and CI clearly